### PR TITLE
Remove 'paster' occurrences and replace with 'ckan'

### LINKED ISSUE #5685: Page View Tracking documentation still refers to paster
**CKAN version**
2.9.1

**Describe the bug**
https://docs.ckan.org/en/2.9/maintaining/tracking.html#enabling-page-view-tracking still refers to `paster`.

**Steps to reproduce**
https://docs.ckan.org/en/2.9/maintaining/tracking.html#enabling-page-view-tracking

**Expected behavior**
Replace `paster` with `ckan`.

### PR DESCRIPTION
**Pull request by tino097**
Fixes #5685 

### Proposed fixes:

Replaced `paster` with `ckan`, few other occurrences 

Change the CLI documentation from `ckan server` to `ckan run` as there are examples with the `ckan run ` command

### Features:

- [ ] includes tests covering changes
- [x] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Review by kowh-ai — APPROVED** [review]
Documentation changes look good Konstantin - thank you.

File: doc/maintaining/cli.rst [hunk 1]
```diff
@@ -482,6 +482,19 @@ The result is saved in profile.data.search. To view the profile in runsnakerun::
...
+run: Start Development server
+==================================
+
+Usage
+
+.. parsed-literal::
+
+ ckan run --host (-h)          - Set Host
+ ckan run --port (-p)          - Set Port
+ ckan run --reloader (-r)      - Use reloader
```
Comment (amercader) [inline doc/maintaining/cli.rst:494]: This syntax changed in https://github.com/ckan/ckan/pull/5639 (we have `--disable-reloader` now). Can you update @tino097 ? cheers

Comment (tino097) [reply doc/maintaining/cli.rst]: Done!

**Comment by tino097** [comment]
this one is good to go @amercader @kowh-ai

**Pull request merged**